package.json

```diff
@@ -18,8 +18,18 @@
     "code",
     "sharing"
   ],
+  "jest": {
+    "setupFiles": [
+      "./tests/testSetup.js"
+    ],
+    "moduleNameMapper": {
+      "^.+\\.(css|styl|svg)$": "identity-obj-proxy"
+    }
+  },
   "dependencies": {
     "codemirror": "^5.33.0",
+    "enzyme": "^3.3.0",
```

Comment:
I believe `enzyme` should be in `devDependencies`, shouldn't it?

Reply:
As well as `enzyme-adapter-react-16`.

Reply:
I also think so, but our linter doesn't

Reply:
No way. Just tried to move it under `devDependencies` - everything works fine.

Reply:
strange things happened